packages/components/src/Tiles/Tiles.tsx

```diff
@@ -3,25 +3,43 @@ import { ReactNode } from 'react';
 import { GapSpaceProp, cn, createVar, gapSpace } from '@marigold/system';
 
 export interface TilesProps extends GapSpaceProp {
+  /**
+   * The children of the component.
+   */
   children: ReactNode;
+
+  /**
+   * Set minimum width for all items inside.
+   * @default '250px'
+   */
   tilesWidth: string;
+
+  /**
+   * Tiles will stretch to available width and will distribute their width equally. Note that this can make them wider than the specified tiles width, but not smaller than the given "tilesWidth". Basically, this is full responsive mode.
+   * @default false
+   */
   stretch?: boolean;
+
+  /**
+   * If true, all items will have the size of the biggest item.
+   * @default false
+   */
   equalHeight?: boolean;
 }
 
 export const Tiles = ({
   space = 0,
   stretch = false,
   equalHeight = false,
-  tilesWidth,
+  tilesWidth = '250px',
```

Comment:
why is there a default value?

Reply:
oh, i think i was testing & forgot it :D @sarahgm

Reply:
but i noticed it exists our docs table with default value `250px` idk why @sarahgm